tests/bluetooth/tester/src/btp/btp_bap.h

```diff
@@ -97,16 +99,81 @@ struct btp_bap_broadcast_scan_stop_cmd {
 #define BTP_BAP_BROADCAST_SINK_SYNC		0x0e
 struct btp_bap_broadcast_sink_sync_cmd {
 	bt_addr_le_t address;
-	uint8_t broadcast_id[3];
+	uint8_t broadcast_id[BT_AUDIO_BROADCAST_ID_SIZE];
 	uint8_t advertiser_sid;
 	uint16_t skip;
 	uint16_t sync_timeout;
+	bool past_avail;
```

Comment:
w.r.t. `bool` vs `uint8_t` that was a question and not a request for change :D 

Does the BTP support using `bool`? If so, I think we have quite a few commands and events that could benefit from moving from `uint8_t` to `bool`. 

Since `bool` is not necessarily a well defined type in all case, it's also understandable if we kept using `uint8_t` as that is well defined.

Reply:
In BTP spec the bool variable is defined as 1 octet and the zephyr tester is built with bool as 1 octet, so it's fine either way.

Reply:
In that case then I'd suggest to either use `bool` for all boolean values (looks like this PR adds a few other still defined as `uint8_t`, or keep using `uint8_t` for all bools and update it later if wanted.

Reply:
lets keep it as uint8_t :-)